### DISCUSSION TITLE
Add support to Kubernetes/Helm

### DISCUSSION THREAD

**Discussion by pierre-geeraert** [jakeswenson/BitBetter#198] [Ideas]
Hello, 
thanks for this software , I used it since a while now.
I recently migrated my docker bitwarden to be used in a Kubernetes environment.
After some struggle mostly due to the secret used to encrypt/decrypt the cert , I finally did succeed to make it working.

The steps are really similar to the README for docker but some steps needs to be taken in advance

I was wondering if this repo would be interested for me to edit the README and add a section or even just to provide the guidance over this discussion.

**Comment by captainhook** [comment]
@pierre-geeraert definitely, you are most welcome to contribute.

**Comment by ringnutz** [comment]
> Hello, thanks for this software , I used it since a while now. I recently migrated my docker bitwarden to be used in a Kubernetes environment. After some struggle mostly due to the secret used to encrypt/decrypt the cert , I finally did succeed to make it working.
> 
> The steps are really similar to the README for docker but some steps needs to be taken in advance
> 
> I was wondering if this repo would be interested for me to edit the README and add a section or even just to provide the guidance over this discussion.

I would also be interested in this information, as my use case is the same.
